__exekutir__/roles/kommandir_workspace_setup/tasks/main.yml

```diff
@@ -123,8 +123,9 @@
   git:
     repo: "{{ item.repo }}"
     dest: "{{ kommandir_workspace }}/cache/{{ item.cachepath }}"
-    # Shallow clone is much faster, two commits will cover a merge-commit
-    depth: "{{ item.depth | default(2) }}"
+    # Shallow clone is much faster, two commits will cover a merge-commit and will be set per repo.
+    # default(omit) to workaround failure with python-bugzilla repo
```

Comment:
This comment about specific failure doesn't belong in general-purpose code.  How about:

```
# Shallow clone is much faster when specific branch/tag/version is _not_ used.
# Otherwise, if a version is specified, then the full history-depth is likely required to find it
```